test/views/payments/OpenPaymentsViewSpec.scala

```diff
@@ -278,17 +279,20 @@ class OpenPaymentsViewSpec extends ViewBaseSpec {
 
   "Rendering the open payments page" should {
```

Comment:
Another absolutely **tiny** thing but could you just reword this a little so it makes grammatical sense (sorry!)